### DISCUSSION TITLE
iCub Software 1.10.0 Released

### DISCUSSION THREAD

**Discussion by mbrunettini** [robotology/community#278] [Releases]
iCub 1.10.0 was released today and is now available for download at: https://github.com/robotology/icub-main/releases/. This release is a feature release and introduces several new features and important changes since iCub 1.8.0

**Comment by drdanz** [comment]
Please note that all the iCub related repositories are tagged as `v1.10.0` but there was an issue with the robots-configuration repository, therefore the correct tag for that repo is `v1.10.0.1`.
